doc/whats_new/upcoming_changes/sklearn.linear_model/32659.api.rst

```diff
@@ -0,0 +1,13 @@
+- Parameter `penalty` of :class:`linear_model.LogisticRegression` and
+  :class:`linear_model.LogisticRegressionCV` is deprecated and will be removed in
+  version 1.10.
+  Simply use `l1_ratio=0` (or `l1_ratios=(0,)`) instead of `penalty="l2"`, `l1_ratio=1`
```

Comment:
One of my pet peeve, I don't see the point of saying "simply do this" or "just do that" the only thing that can happen is that you make the person feel bad if it's not clear enough 😉

```suggestion
  Use `l1_ratio=0` (or `l1_ratios=(0,)`) instead of `penalty="l2"`, `l1_ratio=1`
```